src/style-keeper.js

```diff
@@ -1,11 +1,11 @@
 /* @flow */
 
 export default class StyleKeeper {
-  _userAgent: string;
+  _userAgent: string | typeof undefined;
```

Comment:
was this not being enforced by previous versions of flow or is this a functionality change?

Reply:
Wasn't enforced the way the typings were previously setup, but definitely the same functionality! Verified in prod with a production app we have running that it is initialised with `undefined`.